source/usr/lib/mcvirt/config_file.py

```diff
@@ -65,7 +66,7 @@ def _writeJSON(data, file_name):
         config_file.close()
 
         # Check file permissions, only giving read/write access to libvirt-qemu/root
-        os.chmod(file_name, 0600)
+        os.chmod(file_name, 0o600)
```

Comment:
This should use the defines documented here https://docs.python.org/2/library/os.html#os.chmod rather than a magic number